src/Symfony/Component/DependencyInjection/Loader/Configurator/EnvConfigurator.php

```diff
@@ -31,6 +31,18 @@ public function __toString()
         return '%env('.implode(':', $this->stack).')%';
     }
 
+    /**
+     * @return $this
+     */
+    public function __call(string $name, array $arguments): self
+    {
+        $processor = strtolower(preg_replace(['/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'], '\1_\2', $name));
+
+        $this->custom($processor, ...$arguments);
```

Comment:
let's remove custom() instead, don't you think?

Reply:
I would prefer to use `custom` over magic methods.